ibis/backends/tests/test_generic.py

```diff
@@ -1898,3 +1892,77 @@ def test_subsequent_overlapping_order_by(con, backend, alltypes, df):
     result = con.execute(ts2)
     expected = df.sort_values("id", ascending=False).reset_index(drop=True)
     backend.assert_frame_equal(result, expected)
+
+
+@pytest.mark.broken(
+    ["druid"],
+    raises=PyDruidProgrammingError,
+    reason=(
+        "Query could not be planned. SQL query requires ordering a table by time column"
+    ),
+)
+@pytest.mark.broken(
+    ["dask"],
+    raises=AssertionError,
+    reason="dask doesn't support deterministic .sort_values()",
```

Comment:
We can resolve this by rewriting the expression similarly to the SQL backends. Actually this should be done in a global simplification step implemented in `ibis/expr/rewrites.py`, I am planning to do that in a follow-up.